lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -422,35 +429,35 @@
           - Note that C(npipe) is only supported by Docker for Windows.
         type: str
         choices:
-          - 'bind'
-          - 'volume'
-          - 'tmpfs'
-          - 'npipe'
+          - bind
```

Comment:
it is a good idea to put elements of list alphabetically

Reply:
Done.